src/app/store/accounts/submitted-transactions.ts

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'jotai';
+
+export interface SubmittedTransaction {
+  rawTx: string;
+  txid: string;
+}
+
+export const accountSubmittedTransactionsState = atom<SubmittedTransaction[]>([]);
```

Comment:
I think it does make sense to use an object keyed by txid, rather than an array. This avoids the need to filter duplicates/check if a tx of same id exists. If you broadcast twice and it were to have same txid, just update it (no need for additional handling)

Reply:
Ok, do you think we need to keep the timestamp for any reason?

Reply:
As long as we're not persisting it long term, I don't reckon so

Reply:
Thinking about this, I guess I don't understand needing to use an object. We are NOT doing checks to filter duplicates in the array itself ...we are checking account transactions (confirmed/mempool) to move it in/out of the submitted state if needed. Am I understanding your comment correctly?